### DISCUSSION TITLE
How hard would it be to have a variable depreciation rate?

### DISCUSSION THREAD

**Discussion by Zitchas** [endless-sky/endless-sky#7902] [Balance, Mechanics, and Interactions]
Just thinking about depreciation, and overall it feels much too fast, and personally feels like it is encouraging me to dump my purchased ship at the very first available opportunity. If the time comes when I'm thinking of selling, then do it now, because I'm losing value by the bucketload every day.

So my thought is: Could we implement some kind of variable depreciation rate? have it start after the 7 day grace period, like it is now, but have it start pretty steep: Losing maybe 10% of the ship value within the first month. But then have it soften out a lot so that it takes about our current time-to-depreciate to get down to 60%, and then really flatten out so that it takes 4-5x our current max depreciation time to reach 30%. 

I'm also of the opinion that purchased ships should have a lower bound on depreciation (the 30% mentioned here) that is higher than the 25% depreciation that captured ships start out with. No matter how old my ship gets, it should sell for more than the ship that got shot up to within an inch of its life, soaked in blood (and maybe neurotoxins and/or shrapnel) when it was captured forcibly, and then patched up to get back into service.

**Comment by vitalchip** [comment]
10% in a month is just about what it was/is (there may have been changes from where my code is)

42 days including the 7 grace days takes you to close on 90% of original value, so that's 35 days of depreciation, not far off a month.

It takes about 1000 days to get down to 25% value.

If you wanted a steep initial drop off (a bit like buying a new car where the not newness and taxes generally mean it loses a fair chunk of market value as soon as it is owned) then you could just do something like take 7% of the initial value off as a lump 7 days after purchase.

Then stretch out the existing formula so that at 37 days old it's worth 97% of the value (- the 7% too) and get it to hit 30% value at something around 3000 days (more?)

The problem is that the value already does flatten out and putting the depreciation over a longer time makes the later stages 'flatter'  so 3000 days old is 31.5% of original value and 4000 days is 30%.

This is me thinking of a simple change, a quick addition of the initial drop off (one time hit) and a minor change to a few existing figures. No substantial change to the existing methods.

**Reply by Zitchas** [reply]
Yeah, the new car purchase model was what I was thinking of. Enough of a hit to discourage "temporarily buy a big fleet for one battle and then sell it immediately" sort of behavior, but really soften/minimize the financial loss of sticking with a single ship for several years.

**Comment by vitalchip** [comment]
I think it would probably be an improvement.

For me, I'd like to see depreciation be linked to the player's actions. If you buy a freighter and ply it back and forth leisurely along a safe trade route the depreciation should be much less than a similar ship constantly run ragged through pirate systems getting the shit beaten out of the hull day after day.

**Reply by Zitchas** [reply]
That would be great. Especially if depreciation could be a) tied to *hull* damage, and b) if there was some way to repair/maintain/fix it. I like keeping my ships pristine!

**Comment by vitalchip** [comment]
I think (not having been through all the ups and downs) I would move to a shields/armour/hull system. Both shields and armour being 'consumables' in a sense and hull damage being something far more serious. Not irreversible but something which if repeatedly damaged over time becomes intrinsically weaker (so worth less) and that could have some repercussions beyond money.

I'm not sure exactly what those might be or exactly how they could be implemented but the sort of thing might be loss of ability to carry as much armour on the frame, reduced internal space (outfit/cargo) due to the need for additional bracing (which you need to buy) or limits on acceleration/turning due to stress factors or even in bad cases the frame being too compromised to use hyperspace.

Some/all of these might be a bit much for a game like this but maybe some derivative of the general concept might fly.
